crawler/features_crawler.py

```diff
@@ -1298,6 +1301,42 @@ def _crawl_cpuHw(self):
         feature_key = '{0}-{1}'.format('cpu', cpu.cpu_vendor_id)
         yield (feature_key, cpu)
 
+
+    def crawl_gpu(self):
```

Comment:
but features_crawler.py does not exist any more! In the new plugin model, assuming this plugin is for containers, this should be inside crawler/plugins/gpu_container_crawler.py alongwith a gpu_container_crawler.plugin descriptor.